assets/scripts/script.js

```diff
@@ -1,86 +1,12 @@
-        const galleryaSlider = $(this).closest('.gallerya--slider, .gallerya--product-variation-slider');
-        const navigation = galleryaSlider.data('gallerya-navigation');
-        const pageDots = galleryaSlider.data('galleryaPageDots');
-        const thumbnails = galleryaSlider.find('.js-gallerya-thumbnail-slider');
-        const count = galleryaSlider.find('[data-gallerya-count]');
-        const productLink = galleryaSlider.parents('a.woocommerce-loop-product__link');
-        const sliderArgs = {
-          cellAlign: 'left',
-          contain: true,
-          wrapAround: true,
-          imagesLoaded: true,
-          watchCSS: true,
-          lazyLoad: true,
-        };
-
-        if (!galleryaSlider.hasClass('gallerya--product-variation-slider')) {
-          sliderArgs.arrowShape = arrowShape;
-        }
-        if (typeof pageDots !== 'undefined') {
-          // Let the pageDots property be overriden by a data-attribute.
-          sliderArgs.pageDots = pageDots == true;
-        }
-        else if (typeof navigation === 'undefined' || navigation == false || thumbnails.length > 0) {
-          sliderArgs.pageDots = false;
-        }
-
-        // Adjust styling before slider init.
-        $(this).addClass('flickity');
-        $(this).flickity(sliderArgs);
-
-        // Handle the slide changing to update the variant url
-        $(this).on('change.flickity', function (event, index) {
-          const variantURL = $(this).find('li.is-selected').data('gallerya-variant-permalink');
-          if (variantURL) {
-            productLink.prop('href', variantURL);
-          }
-        });
-
-        const flickityData = $(this).data('flickity');
-        if (thumbnails.length > 0) {
-          const thumbnailsArgs = {
-            asNavFor: element,
-            contain: true,
-            pageDots: false,
-            imagesLoaded: true,
-            groupCells: true,
-            arrowShape: arrowShape,
-          };
-          thumbnails.flickity(thumbnailsArgs);
-
-          $(this).on('select.flickity', function () {
-            const index = flickityData.selectedIndex;
-            const className = 'is-currently-selected';
-            thumbnails.find('.flickity-slider li').removeClass(className)
-              .eq(index).addClass(className);
-          });
-        }
-        if (count) {
-          $(this).on('select.flickity', function () {
-            const slideNumber = flickityData.selectedIndex + 1;
-            count.text(slideNumber + '/' + flickityData.slides.length);
-          });
-        }
-      });
-
-      $('.woocommerce-loop-product__link').on('click', function (e) {
-        // Prevent clicks onto slider arrows to bubble through to wrapping product link.
-        if ($(e.target).closest('.flickity-prev-next-button').length !== 0){
-          e.stopPropagation();
-          return false;
-        }
-      });
-    }
-
+    // Reinitialize slider for Woocommerce Related Accessories
```

Comment:
The event actually doesn't seem to be related to related-accessories, so we should drop its name from the comment here.